src/client/pfs_file.go

```diff
@@ -694,3 +722,22 @@ func (c APIClient) WalkFile(commit *pfs.Commit, path string, cb func(*pfs.FileIn
 		}
 	}
 }
+
+// This is duplicated from the object package to remove a circular dependency.
+func withPipe(wcb func(w io.Writer) error, rcb func(r io.Reader) error) error {
```

Comment:
We should refactor this into a utility package.

Reply:
Really the object package shouldn't depend on the client.

Reply:
It shouldn't be in the client. It should be in a separate utility package that both the client and obj package can import and use.

Reply:
What should we call the utililty package?

Reply:
Hmm, none of the packages we currently have really jump out as the right one to put it in. We could just put it in a package called pipeutil or maybe a package called goutil to meme on Go stuff we need utilities for lol.